src/services/ipfs-service.ts

```diff
@@ -155,4 +166,39 @@ export class IpfsService implements IIpfsService {
 
     return anchorCid
   }
+
+  async importCAR(car: CAR, options: AbortOptions = {}): Promise<void> {
+    const blocks = []
+    for (const block of car.blocks) {
+      blocks.push(block)
+    }
+    await Promise.all(blocks.map(async block => {
```

Comment:
does this risk overwhelming ipfs with too many simultaneous calls to `block.put`?  Maybe we should batch this or even just do them all one at a time since latency isn't that important here anyway?